### PR TITLE
Add extra bottom margin to the list of payments allocations on the Project Budgeting page

### PR DESCRIPTION
**Pull request by Ovifer13**
**Issue #595**
**Description**
Add extra bottom margin to Allocations in `ProjectPage` so the tile doesn't get cut off with the navigation buttons.

**Proof**
image

### CODE REVIEW

File: src/pages/ProjectDetailPage.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { Route, Redirect } from 'react-router-dom'
 import {
     BottomNavigation,
     BottomNavigationAction,
-    Grid
+    Grid,
+    Box
```
Comment (otech47) [inline src/pages/ProjectDetailPage.js:7]: Box is not needed here? @vdegandesa

File: src/pages/ProjectDetailPage.js [hunk 2]
```diff
@@ -32,7 +33,12 @@ class ProjectDetailPage extends React.Component {
 
         return (
             <Grid container justify='center' className='ProjectDetailPage'>
-                <Grid item xs={10} md={6}>
+                <Grid 
+                    item 
+                    xs={10} 
+                    md={6} 
+                    className='GridProjectDetailPage'
```
Comment (otech47) [inline src/pages/ProjectDetailPage.js:40]: When adding class names to DOM elements that aren't a parent container for a React component, we should use a different naming convention so it doesn't cause confusion reading SCSS files that might refer to a capitalize camelcase name 

Suggestion: rename to kebab-case `grid-project-detail-page` or `grid-container`

**Review by otech47 — CHANGES_REQUESTED** [review]

File: src/styles/ProjectDetailPage.scss [hunk 3]
```diff
@@ -5,6 +5,9 @@
         width: 100%;
         z-index: 2
     }
+    .GridProjectDetailPage {
+        margin-bottom: 2rem;
+    }
```
Comment (otech47) [inline src/styles/ProjectDetailPage.scss:10]: Suggestion:

```
> .MuiGrid-item {
    margin-bottom: 2rem;
}
```

**Review by otech47 — CHANGES_REQUESTED** [review]

File: src/styles/ProjectDetailPage.scss (hunk 3 above)
Comment (Ovifer13) [reply src/styles/ProjectDetailPage.scss:10]: If we use `.MuiGrid-item` this happens 
image

**Review by otech47 — APPROVED** [review]

**Pull request merged**